usecases/mail-sender/src/main/java/com/oracle/cloud/sdk/samples/MailSenderController.java

```diff
@@ -0,0 +1,37 @@
+package com.oracle.cloud.sdk.samples;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class MailSenderController {
+	
+	@Autowired
+	MailSenderService service;
+	
+	@RequestMapping(value="/hello")
+	public String hello() {
+		return "Hello World";
+	}
+	
+	@RequestMapping(value="/send", method = RequestMethod.GET)
+	public String send() {
+		service.sendMail(new MailSenderService.MailContent("<YOUR_MAIL_ID>", "Hello from OCI SpringBoot", "Test Message"));
+		return "Done";
```

Comment:
Is there error handling missing here?  See note below in this same file.

Reply:
Yes. error handling has been added.